jina/executors/indexers/__init__.py

```diff
@@ -176,15 +176,15 @@ def write_handler(self):
             return r
 
     def get_query_handler(self):
-        """Get a *readable* index handler when the ``index_abspath`` already exist, need to be overrided"""
+        """Get a *readable* index handler when the ``index_abspath`` already exist, need to be overridden"""
```

Comment:
just curious, should it be `overwritten`? checkout [this link](https://english.stackexchange.com/questions/88400/when-to-use-override-and-overwrite)

Reply:
You override a method, not overwrite it

Reply:
https://stackoverflow.com/a/30330202  https://stackoverflow.com/a/8962940